data-finding/src/DataReader.py

```diff
@@ -4,176 +4,121 @@
 import json
 import csv
 import os
-import datetime
-
-class FileReader:
```

Comment:
Docstrings?

https://www.jetbrains.com/help/pycharm/using-docstrings-to-specify-types.html#example

Reply:
with the docstring ex here you don't need to have the line with types since you're using typehints (fairly) consistently.